controllers/phish.go

```diff
@@ -39,6 +39,8 @@ func CreatePhishingRouter() http.Handler {
 	router.HandleFunc("/track", PhishTracker)
 	router.HandleFunc("/robots.txt", RobotsHandler)
 	router.HandleFunc("/{path:.*}/track", PhishTracker)
+	router.HandleFunc("/{path:.*}/report", PhishReporter)
+	router.HandleFunc("/report", PhishReporter)
```

Comment:
Great catch! I likely would have missed that completely.

Reply:
I'd noticed the weird functionality and have recently been making a web app of my own in Go, so that one jumped out at me when looking at the handlers.